cirq/contrib/paulistring/convert_to_clifford_gates.py

```diff
@@ -89,9 +88,9 @@ def _convert_one(self, op: ops.Operation) -> ops.OP_TREE:
             return op
 
         # Single qubit gate with known matrix?
-        mat = self.extensions.try_cast(ops.KnownMatrix, op)
+        mat = protocols.unitary(op, None)
         if mat is not None and len(op.qubits) == 1:
```

Comment:
Might want to check `len(op.qubits)` first to avoid creating a potentially large matrix unneccessarily:
```python
if len(op.qubits) == 1:
    mat = protocols.unitary(op, None)
    if mat is not None:
        ...
```
Note this sort of thing will be much nicer once [pep 572](https://www.python.org/dev/peps/pep-0572/) is in:
```python
if len(op.qubits) == 1 and (mat := protocols.unitary(op, None)) is not None:
    # do something with mat
```
We just have to wait for python 3.8!

Reply:
Done.